src/webextension/list/manage/components/account-summary.css

```diff
@@ -109,12 +109,33 @@
   list-style-type: none;
 }
 
-.dropdown > ul > li {
-  padding: 8px;
-}
-
 .dropdown-link {
   color: #0c0c0d;
+  display: block;
+  width: 100%;
```

Comment:
With this and the other sizing changes, I think we'll want to revisit what the focus-ring should look like for these elements. It looks kind of broken right now... (to see them in action, just open the dropdown and hit tab to cycle the focus).